core/controllers/editor.py

```diff
@@ -39,6 +39,7 @@
 from core.domain import user_services
 from core.domain import value_generators_domain
 from core.platform import models
+from constants import CONSTANTS
```

Comment:
We generally only import modules (e.g. user_services), so technically this should be `import constants` and we should refer to `constants.CONSTANTS`. That said, do you know if there's a way to have the constants declared as module-level constants instead?